sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-config-topics.adoc

```diff
@@ -0,0 +1,28 @@
+= SAP SuccessFactors - Additional Configuration Information - Mule 4
+
```

Comment:
Set additional configuration for Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector), such as using a query as a filter in Anypoint Studio.